docs/modules/kaolin.ops.spc.rst

```diff
@@ -0,0 +1,151 @@
+.. _kaolin.ops.spc:
+
+kaolin.ops.spc
```

Comment:
Some general tips:
- You can make a list using the `*` character
- You can reference specific part of the whole documentation with `:ref:`, an attribute / variable with `:attr:`, a function with `:func:`, that can create an hyperlink but also slightly change the font
- You can make the font **bold** with `**` or *italic* with `*`
- When naming a variable or some code you can use `` to highlight it.

You can look at the way I used those in: https://raw.githubusercontent.com/NVIDIAGameWorks/kaolin/master/docs/modules/kaolin.ops.batch.rst and the resulting generated doc in: https://kaolin.readthedocs.io/en/latest/modules/kaolin.ops.batch.html

Reply:
I think it's better now